### PR TITLE
add clarifying comments to clustering.py, create exploratory notebook…

### PR DESCRIPTION
**Pull request by jproctor-rebecca**
… for illustrations scraper, add Windows chromedriver for scraper

# Description

Replace with description of PR

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [X] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [X] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

This code is not complete and therefore not a functioning or deployable feature.

### CODE REVIEW

File: notebooks/SS_image_scraper_exploratory_notebook_RJProctor.ipynb [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+        "3. create function/loop to work with label grab"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "TumSwomNrZr4"
+      },
+      "source": [
+        "There are images hidden inside a table that can be seen, but accessed using Beautiful Soup alone.  To overcome this obstacle, I will use a python package that automates a web browser, Selenium.\r\n"
```
Comment (EEdwardsA) [inline notebooks/SS_image_scraper_exploratory_notebook_RJProctor.ipynb]: I can't see the rest of this line; could you insert line break(s)?

File: notebooks/SS_image_scraper_exploratory_notebook_RJProctor.ipynb [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+      ],
+      "execution_count": 1,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "PfsINlO7bhwZ"
+      },
+      "source": [
```
Comment (EEdwardsA) [inline notebooks/SS_image_scraper_exploratory_notebook_RJProctor.ipynb]: Could you insert line breaks here so that the full text is readable?

**Review by EEdwardsA — COMMENTED** [review]
Looks good to me; I've added a couple comments where I can't see the full text

File: notebooks/SS_image_scraper_exploratory_notebook_RJProctor.ipynb (hunk 1 above)
Comment (jproctor-rebecca) [reply notebooks/SS_image_scraper_exploratory_notebook_RJProctor.ipynb]: This is markdown and auto-formats to the size of the screen when viewing in the notebook.  Unfortunately, when reviewing in GitHub, we have to scroll (as far as I know).

File: notebooks/SS_image_scraper_exploratory_notebook_RJProctor.ipynb (hunk 2 above)
Comment (jproctor-rebecca) [reply notebooks/SS_image_scraper_exploratory_notebook_RJProctor.ipynb]: This is markdown and auto-formats to the size of the screen when viewing in the notebook.  Unfortunately, when reviewing in GitHub, we have to scroll (as far as I know).

**Review by James-Hagerman — APPROVED** [review]
Looks great, Thank you for clarifying with the correct terms.

**Review by EEdwardsA — APPROVED** [review]

**Pull request merged**